_posts/2022-05-23-release-kubernetes-operator-1.0.0.md

```diff
@@ -0,0 +1,123 @@
+| Namespace | FlinkDeployment.Count          | Number of managed FlinkDeployment instances per namespace                                                                                                   | Gauge |
+| Namespace | FlinkDeployment.&lt;Status&gt;.Count | Number of managed FlinkDeployment resources per &lt;Status&gt; per namespace. &lt;Status&gt; can take values from: READY, DEPLOYED_NOT_READY, DEPLOYING, MISSING, ERROR | Gauge |
+| Namespace | FlinkSessionJob.Count          | Number of managed FlinkSessionJob instances per namespace                                                                                                   | Gauge |
+
+## What's Next?
+
+Our intention is to advance further on the [Operator Maturity Model](https://operatorframework.io/operator-capabilities/) by adding more dynamic/automatic features
+
+ * Standalone deployment mode support [FLIP-225](https://cwiki.apache.org/confluence/display/FLINK/FLIP-225%3A+Implement+standalone+mode+support+in+the+kubernetes+operator)
+ * Auto-scaling using Horizontal Pod Autoscaler
+ * Dynamic change of watched namespaces
+ * Pluggable Status and Event reporters (Making it easier to integrate with proprietary control planes)
+ * SQL jobs support
+
+## Release Resources
+
+The source artifacts and helm chart are now available on the updated [Downloads](https://flink.apache.org/downloads.html)
+page of the Flink website.
+
+The [official 1.0.0 release archive](https://archive.apache.org/dist/flink/flink-kubernetes-operator-1.0.0/) doubles as a Helm repository that you can easily register locally:
+
+{% highlight bash%}
+$ helm repo add flink-kubernetes-operator-1.0.0 https://archive.apache.org/dist/flink/flink-kubernetes-operator-1.0.0/
+$ helm install flink-kubernetes-operator flink-kubernetes-operator-1.0.0/flink-kubernetes-operator --set webhook.create=false
+{% endhighlight%}
+
+You can also find official Kubernetes Operator Docker images of the new version on [Dockerhub](https://hub.docker.com/r/apache/flink-kubernetes-operator).
+
+For more details, check the [updated documentation]({{site.DOCS_BASE_URL}}flink-kubernetes-operator-docs-release-1.0/) and the
+[release notes](https://issues.apache.org/jira/secure/ReleaseNote.jspa?projectId=12315522&version=12351500).
+We encourage you to download the release and share your feedback with the community through the [Flink mailing lists](https://flink.apache.org/community.html#mailing-lists)
+or [JIRA](https://issues.apache.org/jira/issues/?jql=project%20%3D%20FLINK%20AND%20component%20%3D%20%22Kubernetes%20Operator%22).
+
+## List of Contributors
+
+The Apache Flink community would like to thank each and every one of the contributors that have made this release possible:
+
+Aitozi, Biao Geng, FuyaoLi2017, Gyula Fora, Hao Xin, Jaganathan Asokan, MartijnVisser, Marton Balassi, Matyas Orhidi, Márton Balassi, Nicholas Jiang, SteNicholas, Thomas Weise, Yang Wang, jbusche, lz, ted chang
```

Comment:
We should double check the list before publishing

Reply:
Of cause, I am doing this now.